Common/cpp/LayoutAnimations/LayoutAnimationsManager.h

```diff
@@ -61,6 +63,7 @@ class LayoutAnimationsManager {
   std::unordered_map<int, std::string> viewsScreenSharedTagMap_;
 #endif
 
+  std::unordered_set<int> disableExiting_;
```

Comment:
I renamed it to `shouldAnimateExitingForTag_` since now it is a map, that stores bool values. This allows us to nest config components to better control which animations should be removed (e.g. by setting `skipExiting` to `false` on a subtree).